docs/src/helpers/tokens-helper.js

```diff
@@ -99,6 +99,15 @@ export const getTokens = (tokens, keys = null) =>
       if (token.name.indexOf('_XXL') !== -1) {
         return 7;
       }
+      if (token.name.indexOf('_XXXL') !== -1) {
```

Comment:
We should have a helper function somewhere for this. Something like `includes(token.name, '_XXXL')`. But that's a separate issue!